game/g_session.cpp

```diff
@@ -121,6 +121,8 @@ result session::init (string::view cmdline)
 
     // sound indices are shared over the network so sounds
     // need to be registed in the same order on all clients
+
+#if defined(_WIN32)
```

Comment:
Still need this after stubbing the return value for `cSound::load_sound`?

Reply:
Nope! Removed.